Looks like something is breeaking restoration of `RepoTasks.csproj`:

``` text
  Determining projects to restore...
MSBUILD : error MSB4017: The build stopped unexpectedly because of an unexpected logger failure.
Microsoft.Build.Exceptions.InternalLoggerException: The build stopped unexpectedly because of an unexpected logger failure.
 ---> System.InvalidOperationException: Operation is not valid due to the current state of the object.
   at Microsoft.Build.Collections.RetrievableEntryHashSet`1.Enumerator.MoveNext()
   at Microsoft.Build.BackEnd.Logging.ProjectLoggingContext.FilterEnvironmentDerivedProperties(PropertyDictionary`1 properties)+MoveNext()
   at System.Linq.Enumerable.TryGetLast[TSource](IEnumerable`1 source, Func`2 predicate, Boolean& found)
   at System.Linq.Enumerable.LastOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at Microsoft.DotNet.ArcadeLogging.PipelinesLogger.OnProjectStarted(Object sender, ProjectStartedEventArgs e) in /_/src/Microsoft.DotNet.ArcadeLogging/PipelinesLogger.cs:line 242
   at Microsoft.Build.BackEnd.Logging.EventSourceSink.RaiseProjectStartedEvent(Object sender, ProjectStartedEventArgs buildEvent)
   --- End of inner exception stack trace ---
   at Microsoft.Build.Exceptions.InternalLoggerException.Throw(Exception innerException, BuildEventArgs e, String messageResourceName, Boolean initializationException, String[] messageArgs)
   at Microsoft.Build.BackEnd.Logging.EventSourceSink.RaiseProjectStartedEvent(Object sender, ProjectStartedEventArgs buildEvent)
   at Microsoft.Build.BackEnd.Logging.EventSourceSink.RaiseAnyEvent(Object sender, BuildEventArgs buildEvent)
```

--- a/global.json
+++ b/global.json
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "7.0.100-preview.7.22355.9"
+    "version": "7.0.100-preview.7.22357.3"
   },
   "tools": {
-    "dotnet": "7.0.100-preview.7.22355.9",
+    "dotnet": "7.0.100-preview.7.22357.3",
     "runtimes": {
       "dotnet/x86": [
         "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"

--- a/global.json
+++ b/global.json
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "7.0.100-preview.7.22354.3"
+    "version": "7.0.100-preview.7.22355.9"
   },
   "tools": {
-    "dotnet": "7.0.100-preview.7.22354.3",
+    "dotnet": "7.0.100-preview.7.22355.9",
     "runtimes": {
       "dotnet/x86": [
         "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"

